### ISSUE TITLE
MIN_VOLUME_TO_BUY appears to also be MAX

### ISSUE THREAD

**Issue by utdrmac** [GuntharDeNiro/BTCT#65]
Relevant config section:

```
"BTC_DOGE":  { "strategy": "stepgain", "override": { "MIN_VOLUME_TO_BUY": 1000.0, "MIN_VOLUME_TO_SELL": 1000.0 }}
```

Bot just purchased exactly 1000 DOGE, spending 0.00043000BTC in doing so. BTC_TRADING_LIMIT for stepgain is 0.01

So it appears that these MIN values also set maximums as well which is incorrect behavior. The bot should have bought, at minimum, 1000 DOGE but should have bought up to 0.01 BTC worth.

**Comment by GuntharDeNiro** [comment]
When we send an order to order book, we get the quantity we can find available at that price: the bot sends 0.01BTC worth in the order, but if it is not available, we grab what we can.
Thanks!

**Comment by utdrmac** [comment]
@GuntharDeNiro I have 998.50504567 DOGE in my account. Bot refusing to place buy order. I have MIN TO BUY / SELL set to 1000. Shouldn't the bot ignore the 998.5 as "dust" and continue on?

```
------------------------------------------------------------------------------------------------
 ɢՍͶ฿Ǿͳ v4.0.5  Scanning poloniex for trading opportunities 2017/09/03 15:11:03
------------------------------------------------------------------------------------------------
2017/09/03 15:11:03: BTC_DOGE Last Bid/Ask 4.7e-7/4.8e-7
2017/09/03 15:11:04: Got Trading History
2017/09/03 15:11:05: Averaged bought price: 4.3e-7
2017/09/03 15:11:05: Price to sell: 4.3688e-7
2017/09/03 15:11:05: EMA1: 0.00000049/EMA2: 0.00000049
2017/09/03 15:11:05: Price to buy: 4.811799999999999e-7
2017/09/03 15:11:05: Got Open Orders
2017/09/03 15:11:05: Last Bid/Price to sell: 4.7e-7/4.3688e-7
2017/09/03 15:11:05: Last Ask/Price to buy: 4.8e-7/4.811799999999999e-7
2017/09/03 15:11:05: Trend for BTC_DOGE:    DOWN!!!
2017/09/03 15:11:05: We are on level 2 to sell (bought/sell at): 4.3e-7/4.3688e-7
 ɢՍͶß๏Ŧ v4.0.5  Tips&Tricks: BTC_DOGE can be sold, checking conditions...
2017/09/03 15:11:05: There are some trading opportunities for BTC_DOGE at poloniex. Scanning Trade History
2017/09/03 15:11:05: Last Bid price BTC_DOGE: 4.7e-7 | Last Ask: 4.8e-7
2017/09/03 15:11:05: Balance BTC: 0.11441466
2017/09/03 15:11:05: Balance DOGE: 998.50504567
2017/09/03 15:11:05: On Orders DOGE: 0.00000000
2017/09/03 15:11:05: Trend for BTC_DOGE:    DOWN!!!
Processing strategy STEPGAIN for BTC_DOGE on poloniex
┌───────────────┬─────────────────────────────────────────────┬────────────┬──────────────────────┬────────┬────────────┬──────────────┬──────────┐
│ ɢՍŋƁʘͳ v4.0.5 │ EMA1                                        │ EMA2       │ Buy at               │ Last   │ BTC        │ DOGE         │ Pair     │
├───────────────┼─────────────────────────────────────────────┼────────────┼──────────────────────┼────────┼────────────┼──────────────┼──────────┤
│ Buy cycle     │ 0.00000049                                  │ 0.00000049 │ 4.811799999999999e-7 │ 4.8e-7 │ 0.11441466 │ 998.50504567 │ BTC_DOGE │
├───────────────┼─────────────────────────────────────────────┼────────────┴──────────────────────┴────────┴────────────┴──────────────┴──────────┤
│ Callbacks     │ Balance is lower than minimum volume to buy │                                                                                   │
└───────────────┴─────────────────────────────────────────────┴───────────────────────────────────────────────────────────────────────────────────┘
```
